samples/ble_gateway/ble_printer/src/ble_printer.c

```diff
@@ -109,8 +114,8 @@ void BLEPrinter_Receive(MODULE_HANDLE module, MESSAGE_HANDLE message)
             const char* source = ConstMap_GetValue(props, GW_SOURCE_PROPERTY);
             if (source != NULL && strcmp(source, GW_SOURCE_BLE_TELEMETRY) == 0)
             {
-                const char* ble_controller_id = ConstMap_GetValue(props, GW_BLE_CONTROLLER_INDEX_PROPERTY);
-                const char* mac_address_str = ConstMap_GetValue(props, GW_MAC_ADDRESS_PROPERTY);
+                //const char* ble_controller_id = ConstMap_GetValue(props, GW_BLE_CONTROLLER_INDEX_PROPERTY);
```

Comment:
Being a sample, its possible that someone would want to use these for diagnostics or whatever. I figured keeping this around could be useful.